Userland/Libraries/LibCore/File.h

```diff
@@ -17,6 +17,9 @@
 
 namespace Core {
 
+#define DEFAULT_PATH "/usr/local/sbin:/usr/local/bin:/usr/bin:/bin"
+#define DEFAULT_PATH_SV "/usr/local/sbin:/usr/local/bin:/usr/bin:/bin"sv
```

Comment:
This feels more like a `Core::FileSystem` thing, I just put it in `Core::DeprecatedFile` originally because we didn't have a better alternative back then (and therefore it was subsequently copied to `FileSystem`).

Reply:
*shrug* Then I'll merge it into FileSystem

Reply:
Ah! No, sorry, there is a reason against this: Sometimes this introduces a dependency on LibFileSystem that wouldn't exist before, so I'd like to avoid that.

Reply:
Hmm. the only example for "reason against this" is `LibC/unistd.cpp`, which already has the line `#include <LibFileSystem/FileSystem.h>`, so that argument is invalid.

Yup, I'll move it to LibFileSystem.